src/webhookValidator.ts

```diff
@@ -14,7 +14,7 @@ export const ERR_TOKEN_REQUIRED =
 function getUrlFromContext(context: Context): string {
   return url.format({
     protocol: context.protocol,
-    host: context.host,
+    host: process.env.HOST || context.host,
```

Comment:
Environment variables are not very friendly for flexibility. Can you move this into an options argument instead?

Reply:
fair point